src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -488,7 +488,7 @@ bool QgsPostgresConn::getTableInfo( bool searchGeometryColumnsOnly, bool searchP
       dimName    = QStringLiteral( "l.coord_dimension" );
       gtableName = QStringLiteral( "geometry_columns" );
     }
-    else if ( i == SctGeography )
+    else if ( i == SctGeography && mPostgresqlVersion >= 150000 )
```

Comment:
testing the postgres version looks wrong to me. Should rather be against mPostgisVersionMajor and mPostgisVersionMinor

Reply:
yes, right. Thanks for noticing.